libs/features/spectrum-web-components/src/theme.js

```diff
@@ -1,5 +1,6 @@
 import '@spectrum-web-components/theme/sp-theme.js';
 import '@spectrum-web-components/theme/theme-light.js';
+import '@spectrum-web-components/theme/theme-darkest.js';
```

Comment:
@Axelcureno please do not update this one which will impact catalog page performance.
src/themes/darkest.js one below should be sufficient.

Reply:
Removed 👍🏼